### PR TITLE
fix: Profile target available to ALL commands

### PR DESCRIPTION
**Pull request by z3z1ma**
# Description

This solves for profile target not being available in all commands and defaulting to the default profile. 

# How was the change tested

I personally ran it on my own dbt projects. 

# Issue Information

Relevant issue #346 

# Checklist

- [x] I formatted my PR name according to [CONTRIBUTING.md](CONTRIBUTING.md)
- [ ] I added a news fragment to help populating the changelog as encouraged in [CONTRIBUTING.md](CONTRIBUTING.md)
- [x] I added "Closes #<issue_number>" in the "Issue Information" section (if no issue, feel free to tick thick the box anyway).

### CODE REVIEW

**Review by z3z1ma — COMMENTED** [review]
Black formatted mine a little different, interesting. Looks good though.

File: dbt_sugar/core/task/base.py [hunk 1]
```diff
@@ -49,7 +49,9 @@ def __init__(
         self.dbt_tests: Dict[str, List[Dict[str, Any]]] = {}
         self.build_descriptions_dictionary()
 
-    def get_connector(self) -> Union[PostgresConnector, SnowflakeConnector, RedshiftConnector]:
+    def get_connector(
```
Comment (bastienboutonnet) [inline dbt_sugar/core/task/base.py:52]: this project allows for a line length of 100 chars. I would rather leave the pre-commit hook take care of the formatting as it will standardise it for everyone. Could you please revert this change?

File: dbt_sugar/core/task/base.py [hunk 2]
```diff
@@ -432,15 +436,23 @@ def find_model_schema_file(self, model_name: str) -> Tuple[Optional[Path], bool,
                         schema_file_exists = False
                         if schema_file_path.exists():
                             schema_file_exists = True
-                        return schema_file_path, schema_file_exists, is_already_documented
+                        return (
```
Comment (bastienboutonnet) [inline dbt_sugar/core/task/base.py:439]: this project allows for a line length of 100 chars. I would rather leave the pre-commit hook take care of the formatting as it will standardise it for everyone. Could you please revert this change?

File: dbt_sugar/core/task/base.py [hunk 3]
```diff
@@ -432,15 +436,23 @@ def find_model_schema_file(self, model_name: str) -> Tuple[Optional[Path], bool,
...
+                        )
 
                     if schema_file_path and model_file_found:
                         logger.debug(
                             f"'{model_name}' found in '{schema_file_path}' we'll update entry."
                         )
                         is_already_documented = True
                         schema_file_exists = True
-                        return schema_file_path, schema_file_exists, is_already_documented
+                        return (
```
Comment (bastienboutonnet) [inline dbt_sugar/core/task/base.py:451]: this project allows for a line length of 100 chars. I would rather leave the pre-commit hook take care of the formatting as it will standardise it for everyone. Could you please revert this change?

File: dbt_sugar/core/main.py [hunk 4]
```diff
@@ -40,7 +40,7 @@ def check_and_print_version() -> str:
     prog="dbt-sugar",
     formatter_class=argparse.RawTextHelpFormatter,
     description="CLI tool to help users document their dbt models",
-    epilog="Select onf of the available sub-commands with --help to find out more about them.",
+    epilog="Select one of the available sub-commands with --help to find out more about them.",
```
Comment (bastienboutonnet) [inline dbt_sugar/core/main.py:43]: 👌

File: dbt_sugar/core/task/base.py [hunk 5]
```diff
@@ -230,11 +234,11 @@ def update_column_description_from_schema(
             for column in model.get("columns", []):
                 column_name = column["name"]
                 if column_name in dict_column_description_to_update:
-                    new_desctiption = dict_column_description_to_update[column_name].get(
+                    new_description = dict_column_description_to_update[column_name].get(
```
Comment (bastienboutonnet) [inline dbt_sugar/core/task/base.py:237]: good catch. I tend to be the master of consistent typos since I use autocomplete so much 😂

File: dbt_sugar/core/task/base.py [hunk 6]
```diff
@@ -213,7 +215,9 @@ def update_model_description_test_tags(
         )
 
     def update_column_description_from_schema(
-        self, path_file: Path, dict_column_description_to_update: Dict[str, Dict[str, Any]]
+        self,
```
Comment (bastienboutonnet) [inline dbt_sugar/core/task/base.py:218]: this project allows for a line length of 100 chars. I would rather leave the pre-commit hook take care of the formatting as it will standardise it for everyone. Could you please revert this change?

**Review by bastienboutonnet — CHANGES_REQUESTED** [review]
Thanks a lot @z3z1ma for your proposed changes they make a ton of sense. 

I don't agree with some of the formatting changes that were applied as they seem to be changes from black with a line length shorter than what this repo allows. I might have to make this clearer or provide some config arguments that black can read from the project if users to use black already so that this doesn't happen. 

I use pre-commit to enforce style so I would prefer if you could revert the formatting changes I labelled, this makes sure that non-user specific black settings are introduced.

Once the formatting changes are fixed, I'll approve and merge this PR.

Could you also write a new fragment to shoutout your contribution by any chance? These instructions should help you: https://github.com/bitpicky/dbt-sugar/blob/main/CONTRIBUTING.md#shout-out-your-hardwork-in-the-changelog make sure you add a "by @z3z1ma " with a markdown link so that when I generate the changelog proper credits can be highlighted.

**Review by bastienboutonnet — APPROVED** [review]
Awesome thanks for all the hard work @z3z1ma. This will be in the next patch release.
